pkg/ytconfig/scheduler.go

```diff
@@ -51,7 +51,9 @@ func getControllerAgentLogging(spec *ytv1.ControllerAgentsSpec) Logging {
 func getSchedulerServerCarcass(spec *ytv1.SchedulersSpec) (SchedulerServer, error) {
 	var c SchedulerServer
 	c.RPCPort = consts.SchedulerRPCPort
-	c.MonitoringPort = *spec.InstanceSpec.MonitoringPort
+	if spec.InstanceSpec.MonitoringPort != nil {
+		c.MonitoringPort = *spec.InstanceSpec.MonitoringPort
+	}
```

Comment:
It seems there are more places like that which lead to panic and go unnoticed since we running ytop in tests inside kind and it restarts it on panic.
https://github.com/ytsaurus/ytsaurus-k8s-operator/issues/446

In which tests it was reproduced so you want and fix this?